lib/adal-aad-node/src/engine/AzureAuth.ts

```diff
@@ -0,0 +1,469 @@
+import { ProviderSettings, SecureStorageProvider, Tenant, AADResource, LoginResponse, Deferred, AzureAccount, Logger, MessageDisplayer, ErrorLookup, CachingProvider, RefreshTokenPostData, AuthorizationCodePostData, TokenPostData, AccountKey, StringLookup, DeviceCodeStartPostData, DeviceCodeCheckPostData, AzureAuthType, AccountType, UserInteraction } from '../models';
```

Comment:
Please add the `Microsoft` license header in all the `.ts` files you're checking in